docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+
+## Sessions
+
+| Feature               | Go  | Java | TypeScript | Python | .Net | PHP |
+| --------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Sessions](#sessions) | Yes | No   | No         | No     | No   | No  |
+
+The Go SDK provides a support for [Worker Session](dev-guide/go/features#enable-sessions).
+This makes task routing seamless by ensuring that Activity Tasks are dispatched to the same Worker without having to handle the intricacies of Task Queue naming.
+This abstraction simplifies scenarios where consistency or state is pivotal.
+
+While Go provides built-in support, other SDKs like TypeScript and Python don't have a native concept of Sessions.
+Instead, they embrace a design pattern known as [Worker-Specific Task Queues](#worker-specific-task-queues) to achieve similar consistency guarantees.
+Under this model, all computational tasks of a Workflow are designed to execute on a single Worker.
+Although this might seem like a manual replica of the Session feature in Go, it is often the preferred way to ensure tasks are performed in a consistent environment.
+
+For instance, in scenarios where there's a need to interact with file systems—like data processing tasks or engagements with legacy structures—a sticky execution ensures that all tasks are performed in the same file context.
+In the provided example, text files are written to Worker-specific folders.
+While this serves as a demonstration, real-world applications might involve dedicated machines in a Worker cluster, each corresponding to a 'sticky' Worker, or Worker-Specific Task Queue.
+
+## Signal/Cancel External Workflow
+
+Signal/Cancel External Workflow refers to the process of externally requesting the cancellation of a Workflow.
+
+| Feature                                                            | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------------------------------------------------------ | --- | ---- | ---------- | ------ | ---- | --- |
+| [Signal/Cancel External Workflow](#signalcancel-external-workflow) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+Cancellation is an external request to a Workflow from outside, typically through the Client.
+In Temporal, canceling a workflow itself doesn't make sense as cancellation is an external action.
+Instead of using self-cancellation, you should request cancellation through the Client.
+
+Temporal Workflows can wait for external Signals or events before proceeding with execution.
+External Signals can be used to trigger the rerun of a Workflow after a specified period, including the time spent waiting for the Signal.
+
+<!-- https://temporaltechnologies.slack.com/archives/C01FG4BRQVB/p1697069944495119 -->
+
+In Python, the Cancellation and Signalling is done on the [handle of a Workflow](https://github.com/temporalio/sdk-python#external-workflows).
+
+## Static analyzer
```

Comment:
Dev tooling can be very language specific and not deserving of global callout here I don't believe